iOS/Delegates/AppDelegate.swift

```diff
@@ -140,7 +140,81 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIOnboardingViewControlle
+                                let dl = AppDownload()
+                                try handleIPAFile(destinationURL: destinationURL, uuid: uuid, dl: dl)
+                                
+                                DispatchQueue.main.async {
+                                    self.loaderAlert.dismiss(animated: true) {
+                                        let downloadedApps = CoreDataManager.shared.getDatedDownloadedApps()
+                                        if let downloadedApp = downloadedApps.first(where: { $0.uuid == uuid }) {
+                                            let signingDataWrapper = SigningDataWrapper(signingOptions: UserDefaults.standard.signingOptions)
+                                            signingDataWrapper.signingOptions.installAfterSigned = true
+                                            
+                                            let libraryVC = LibraryViewController()
+                                            let ap = SigningsViewController(
+                                                signingDataWrapper: signingDataWrapper,
+                                                application: downloadedApp,
+                                                appsViewController: libraryVC
+                                            )
+                                            
+                                            ap.signingCompletionHandler = { success in
+                                                if success {
+                                                    if let workspace = LSApplicationWorkspace.default() {
+                                                        if let bundleId = downloadedApp.bundleidentifier {
+                                                            workspace.openApplication(withBundleID: bundleId)
+                                                        }
+                                                    }
+                                                    libraryVC.fetchSources()
+                                                    libraryVC.tableView.reloadData()
+                                                }
+                                            }
+                                            
+                                            let navigationController = UINavigationController(rootViewController: ap)
+                                            
+                                            if UIDevice.current.userInterfaceIdiom == .pad {
+                                                navigationController.modalPresentationStyle = .formSheet
+                                            } else {
+                                                navigationController.modalPresentationStyle = .fullScreen
+                                            }
+                                            
+                                            rootViewController.present(navigationController, animated: true)
```

Comment:
It's possible to make it automatically sign instead of needing the user to press a button before (which could fail depending on some circumstances), should this continue to present that sheet?

Reply:
ah good point, what would you prefer? I tend to generally adjust some settings when signing but I would guess most people are fine with default stuff and would probably like it to sign immediately